addons/ui/stringtable.xml

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Make sure this doesn't readd the BOM removed in https://github.com/CBATeam/CBA_A3/pull/1000

Reply:
i've used tabler 0.8.0, which shouldn't.

Reply:
Why is it pink then?

Reply:
good question, no idea :/

Reply:
just checked, no BOM.

Reply:
Just checked #1000 it indeed converted UTF8-BOM to UTF8 in ui stringtable.
Also checked the merge commit. Both files have same encoding.. There was indeed no change here 🤔  @Github broken.

Reply:
However @alganthe changed the encoding from BOM to nonBOM in the strings stringtable which @veteran29 forgot